luci-app-olsrd2/root/etc/config/luci_olsrd2

```diff
@@ -0,0 +1,3 @@
+config 'olsrd2' 'general'
+	option 'resolve' '1'
+	option 'domain' 'olsr'
```

Comment:
This looks to me that it should not be located in LuCI package. Yes, you can find some a few outdated packages with LuCI, which comes with config file, but thats not common these days. Its not marked even as conffile, though.